### PR TITLE
Ternary Expression for boolean states

### LINKED ISSUE #14: Generating ternary expressions
When doing conditional rendering based on a `boolean` state value, we can introduce ternary rendering in JSX:

```javascript
<button onClick={() => setIsExpanded(!isExpanded)}>
  {isExpanded && "Collapse"}
  {!isExpanded && "Expand"}
</button>
```

can be re-written as

```javascript
<button onClick={() => setIsExpanded(!isExpanded)}>
  {isExpanded ? "Collapse" : "Expand"}
</button>
```

### PR DESCRIPTION
**Pull request by JayaKrishnaNamburu**
Added ability to render ternary expressions for boolean

A little bit of cleanup is needed and I am working on here is the [sandbox](https://codesandbox.io/s/jxw2786x5) link where you can refer the ``UIDLSchema`` and code that is generated in ``App.js`` component.

fixes #14

### CODE REVIEW

**Comment by JayaKrishnaNamburu** [comment]
Hi @alexnm can you check this out once, I will work on the codecov in the mean time.

**Comment by alexnm** [comment]
@JayaKrishnaNamburu please remove the WIP prefix if the PR is ready to be reviewed, it's easier to follow this way. will have a look

**Comment by vladnicula** [comment]
I like it, seems a clean approach. I'm not sure in which case we enter the else branch (https://github.com/teleporthq/teleport-code-generators/pull/89/files#diff-3b11760f27eeb3b9b880da57f49b6ffcR203).

**Comment by JayaKrishnaNamburu** [comment]
Cool, will make all those changes 😄

**Comment by JayaKrishnaNamburu** [comment]
Hey @alexnm  can you check the updated changes, I think need to make few changes in the project sample to cover up the 0.2% code coverage too

File: src/shared/utils/ast-jsx-utils.ts [hunk 1]
```diff
@@ -340,3 +340,13 @@ const convertToUnaryOperator = (operation: string): UnaryOperation => {
     return '!'
   }
 }
+
+export const createTernaryOperation = (
```
Comment (alexnm) [inline src/shared/utils/ast-jsx-utils.ts]: all publicly exported functions from ast utils have the last optional param set like this
```t = types```

this is useful for potentially mocking it when unit testing the functions. please add this last param to keep the consistency. it's the only change needed, as this is not supplied from the outside yet

**Comment by alexnm** [comment]
> Hey @alexnm can you check the updated changes, I think need to make few changes in the project sample to cover up the 0.2% code coverage too

we're fine for now, I'm doing some changes in the tests as we speak.

File: src/shared/utils/ast-jsx-utils.ts (hunk 1 above)
Comment (JayaKrishnaNamburu) [reply src/shared/utils/ast-jsx-utils.ts]: Updated 👍

**Comment by JayaKrishnaNamburu** [comment]
Hey @alexnm let me know if there is any priority issue for the Beta, as this PR is about to complete today.

**Pull request merged**